governance/polls/Debt Ceiling Adjustment for ETH-B - November 30, 2020.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Adjust the ETH-B Debt Ceiling - November 30, 2020
+summary: Signal your support or opposition for adjusting the ETH-B Debt Ceiling from 10 million DAI to TODO $new_ceiling million DAI.
+discussion_link: https://forum.makerdao.com/t/signal-request-adjust-eth-b-debt-ceiling/5282
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Adjust the ETH-B Debt Ceiling - November 30, 2020
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the [voting system](https://vote.makerdao.com/polling). The community can vote in this poll to express support or opposition to adjusting the ETH-B Debt Ceiling from 10 million DAI to **TODO** $new_ceiling million DAI.
+
+This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, November 30 at 16:00 UTC.
+
```

Comment:
So I just merged some changes to the templates that it would be good to include. https://github.com/makerdao/community/blob/master/governance/polls/templates/Debt%20Ceiling%20Adjustment%20for%20%24ilk_name%20-%20%24date_MONTH_DD%2C_YYYY.md